rust/src/lightning.rs

```diff
@@ -587,22 +577,6 @@ pub async fn run_ldk(system: &LightningSystem) -> Result<BackgroundProcessor> {
         Some(scorer),
     );
 
-    let peer_data_path = format!("{ldk_data_dir}/channel_peer_data");
-    let mut info = disk::read_channel_peer_data(Path::new(&peer_data_path))?;
-    for (pubkey, peer_addr) in info.drain() {
-        for chan_info in system.channel_manager.list_channels() {
-            if pubkey == chan_info.counterparty.node_id {
-                block_on(async {
-                    let _ = connect_peer_if_necessary(
```

Comment:
❓ The reconnect is handled automatically based on the `channel_peer_data` file? Is this file persisted forever?

I'm a bit confused by the commit message

> I noticed that after combining the connection and run ldk functions, that the connection never got re-established after restarting. 

So, we do reconnect automatically after removing this code?

Reply:
> ❓ The reconnect is handled automatically based on the `channel_peer_data` file? Is this file persisted forever?

yes, it has been written to the file system, however we don't need that, for the reasons described in the commit message.

> So, we do reconnect automatically after removing this code?

yes.